src/OSPSuite.UI/Views/Charts/AxisSettingsView.cs

```diff
@@ -160,24 +172,17 @@ private RepositoryItem getButton(Axis axis)
          switch (axis.AxisType)
          {
             case AxisTypes.X:
-               return new RepositoryItem();
+               return _disableRepositoryItem;
             case AxisTypes.Y:
                return _addButtonRepository;
             default:
                return _deleteButtonRepository;
          }
       }
 
-      public override void Refresh()
-      {
-         base.Refresh();
-         _gridBinderAxes.Rebind();
-      }
-
       public void BindToSource(IEnumerable<Axis> axes)
       {
-         _gridBinderAxes.BindToSource(axes);
-         Refresh();
+         _gridBinderAxes.BindToSource(axes.ToBindingList());
```

Comment:
This is the key. With AutoBind, a binding list (or obervablecolleciton)_ should be implemented so that DevExpress reacts to cahnges in the data source